jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/FinalizingStep.java

```diff
@@ -74,12 +75,13 @@ public Void call() throws ExecutionException {
       asyncDependencies.addSteps(NonBlockingSteps.get(wrappedDependency));
     }
 
-    // TODO: Don't let future error be suppressed
-    asyncDependencies.whenAllSucceed(
-        () -> {
-          buildConfiguration.getEventDispatcher().dispatch(LogEvent.lifecycle("Finalizing..."));
-          return null;
-        });
+    // This suppresses any exceptions of this future.
```

Comment:
You mean "suppresses compile warning"? This reads to me like, by having this, image building exceptions when running the future will not be reported at run time.

Reply:
This is in fact suppressing the exceptions that could happen in this future, but I think it's okay since it's just an extraneous logging message.

Reply:
Okay, just thought the comment does not match the PR title.